hatch/src/generators/tup/builder.rs

```diff
@@ -30,9 +30,7 @@ impl<'builder> Builder<'builder> {
         }
     }
 
-    pub fn collect_assets(builder: Builder) -> Vec<ProjectAsset> {
-        builder.assets
-    }
+    pub fn collect_assets(self) -> Vec<ProjectAsset> { self.assets }
```

Comment:
I definitely agree with the change; but a side note, is collect really the right word here? All this is doing is returning the assets... we could just rename this method to "assets".

Reply:
ok I will call it `assets`